apps/website/.gitignore

```diff
@@ -1,3 +1,6 @@
 # build output
 dist/
 .output/
+
+# Environment variables
+.env
```

Comment:
maybe should be moved to the root gitinore? we always want to ignore `.env`

Reply:
Sounds good, added this to the root .gitignore [`a16f453` (#1365)](https://github.com/iTwin/iTwinUI/pull/1365/commits/a16f453437d50ee185ff0bef1b0a1ff26c373caf)